src/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -86,20 +86,22 @@ internal PublicClientApplicationBuilder WithBroker(bool enableBroker)
             return this;
         }
 
-#if !ANDROID_BUILDTIME && !WINDOWS_APP && !WINDOWS_APP_BUILDTIME && !NET_CORE_BUILDTIME && !DESKTOP_BUILDTIME && !MAC_BUILDTIME
-#if iOS || NETSTANDARD1_3
+#if !ANDROID_BUILDTIME && !WINDOWS_APP_BUILDTIME && !NET_CORE_BUILDTIME && !DESKTOP_BUILDTIME && !MAC_BUILDTIME
```

Comment:
This is ok, but it might be more readable to say if iOS_BUILDTIME || NETSTANDARD_BUILDTIME

Reply:
Yes, but when I did that the api was hidden when i tried to use it in the xamarin shared code. Is that expected?

Reply:
using iOS_BUILDTIME || NETSTANDARD_BUILDTIME actually prevents this api from being accessible in shared projects.